### PR TITLE
chore(InputText): migrate InputText component to belonging

### PR DESCRIPTION
**Pull request by cnairi**
## DESCRIPTION

- Component: https://github.com/WTTJ/welcome-ui/pull/2984/commits/dcd619be9ccd927506eac68df03be6388dded551
- Migration script: https://github.com/WTTJ/welcome-ui/pull/2984/commits/48f1ea0435ef4cc30ac0d468d01cf5ba3c433540
- Actual migration: https://github.com/WTTJ/welcome-ui/pull/2984/commits/202a93ccde36edfb2326f93c7af096e1ae546d78

## HOW TO TEST

Go to https://wui-208-input-text.preview.welcome-ui.com/components/input-text
or start your project locally

## SCREENSHOTS / SCREEN RECORDINGS

Capture d’écran 2025-10-16 à 10 58 03

## COMPATIBILITY

- [x] Tested on Safari (desktop)
- [x] Tested on Chrome (desktop)
- [x] Tested on Firefox (desktop)
- [x] Tested on mobile device sizes
- [x] Tested on tablet device sizes
- [ ] Tested on IOS Safari (either device or simulator)

## QA

- [x] Thoroughly tested in local environment
- [ ] Added tests for all new features
- [ ] Added tests that considered edge cases

### CODE REVIEW

File: lib/src/components/InputText/docs/index.mdx [hunk 1]
```diff
@@ -21,10 +21,9 @@ Property `disabled` can be pass to the component or Field component
 
 Use size property with option:
 
-- `xs` (24px)
-- `sm` (32px)
-- `md` (40px - default)
-- `lg` (48px)
+- `sm` (25px)
+- `md` (33px - default)
```
Comment (theo-mesnil) [inline lib/src/components/InputText/docs/index.mdx]:
```suggestion
- `md` (33px)
```

File: lib/src/components/InputText/index.tsx [hunk 2]
```diff
@@ -66,14 +67,26 @@ export const InputText = forwardRef<HTMLInputElement, InputTextProps>(
...
         {hasLeftIcon ? (
-          <div className={cx('icon-wrapper', `icon-placement-left-${iconSize}`)}>
+          <div
+            className={cx(
+              'icon-wrapper',
+              `icon-placement-left-${iconSize}`,
+              inputProps.disabled && 'disabled'
+            )}
+          >
             {React.cloneElement(icon, { ...icon.props, size: iconSize })}
```
Comment (theo-mesnil) [inline lib/src/components/InputText/index.tsx:77]: Maybe this can work ?
```suggestion
   <Icon size={iconSize} />
```

File: lib/src/components/InputText/input-text.module.scss [hunk 3]
```diff
@@ -23,27 +24,32 @@
...
-      --borderColor: var(--borderColorHover, var(--color-background-neutral-subtle));
+      --borderColor: var(--borderColorHover, var(--color-border-neutral-subtlest));
+      --inputTextBoxShadow: var(--inset-20);
     }
 
     &[data-focus-visible],
     &:focus-visible {
-      --borderColor: var(--borderColorFocused, var(--components-colors-input-color-border-active));
-      outline-color: var(--outlineColorFocus, var(--color-background-brand-subtle));
+      /* --borderColor: var(--borderColorFocused, var(--components-colors-input-color-border-active)); */
```
Comment (theo-mesnil) [inline lib/src/components/InputText/input-text.module.scss]: remove comment

File: lib/src/components/InputText/index.tsx (hunk 2 above)
Comment (cnairi) [reply lib/src/components/InputText/index.tsx:77]: I thought it was working and then my tests broke

It's due to the weird implem of the PasswordInput:
```
  <InputText
        {...rest}
        icon={
          <ToggleButton
            aria-label={toggleAriaLabel ?? (isHidden ? 'Show password' : 'Hide password')}
            data-testid={dataTestId}
            isHidden={isHidden}
            onClick={handleToggle}
          />
        }
        iconPlacement="right"
        ref={ref}
        type={type}
      />
```

See, the icon is a component 🤯 

I will handle that in my next component PR, as I have the PasswordInput assigned to me
